core/src/main/java/org/ehcache/EhcacheManager.java

```diff
@@ -471,6 +471,7 @@ public void init() {
               LOGGER.error("Cache '{}' could not be removed due to ", toBeClosed, exceptionClosingCache);
           }
         }
+        serviceLocator.stopAllServices();
```

Comment:
If this throws, it will shadow the original exception. As a consequence it will make fixing the problem harder.